wpimath/src/test/java/edu/wpi/first/math/kinematics/SwerveDriveKinematicsTest.java

```diff
@@ -406,4 +407,202 @@ void testDesaturateNegativeSpeed() {
         () -> assertEquals(-1.0, arr[2].speed, kEpsilon),
         () -> assertEquals(-1.0, arr[3].speed, kEpsilon));
   }
+
+  @Test
+  void testTurnInPlaceInverseAccelerations() {
+    ChassisAccelerations accelerations = new ChassisAccelerations(0, 0, 2 * Math.PI);
+    double angularVelocity = 2 * Math.PI;
+    var moduleAccelerations =
+        m_kinematics.toSwerveModuleAccelerations(accelerations, angularVelocity);
+
+    //    For turn-in-place with angular acceleration of 2π rad/s² and angular velocity of 2π rad/s,
```

Comment:
You can unindent the comment contents so there's one space between the `//` and the first word.